var/spack/repos/builtin/packages/py-bigfloat/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBigfloat(PythonPackage):
+    """Arbitrary-precision correctly-rounded floating-point arithmetic, via MPFR."""
+
+    homepage = "http://github.com/mdickinson/bigfloat"
+    pypi     = "bigfloat/bigfloat-0.4.0.tar.gz"
+
+    version('0.4.0', sha256='58b96bde872aca5989d13d82eba3acf2aa1b94e22117dd72a16ba5911b0c0cb8')
+
+    depends_on('py-cython', type='build')
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('gmp', type=('build', 'run'))
```

Comment:
`gmp` doesn't have a `prefix.bin` directory, so this dependency currently isn't being used at all by the package